src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +25,20 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryDao countryDao = new CountryDaoImpl(sessionFactory);
+        CountryService countryService = new CountryServiceImpl(countryDao);
```

Comment:
Better to do it like this:

`CountryService countryService = new CountryServiceImpl(new CountryDaoImpl(sessionFactory));`

Also for `ActorService` and `MovieService`.